### PR TITLE
0.11.3

### PR DESCRIPTION
**Pull request by Keats**

### CODE REVIEW

File: src/builtins/filters/array.rs [hunk 1]
```diff
@@ -66,6 +66,29 @@ pub fn sort(value: Value, args: HashMap<String, Value>) -> Result<Value> {
...
+/// Use the `start` argument to define where to start (inclusive, default to `0`)
+/// and `end` argument to define where to stop (exclusive, default to the length of the array)
+/// `start` and `end` are 0-indexed
+pub fn slice(value: Value, args: HashMap<String, Value>) -> Result<Value> {
+    let mut arr = try_get_value!("slice", "value", Vec<Value>, value);
+    if arr.is_empty() {
+        return Ok(arr.into());
+    }
+
+    let start = try_get_value!("slice", "start", f64, args.get("start").unwrap_or(&0.0.into())) as usize;
```
Comment (bootandy) [inline src/builtins/filters/array.rs]: Forgive my curiosity, but why f64 and not an integer?

Comment (Keats) [reply src/builtins/filters/array.rs]: See https://github.com/Keats/tera/issues/254#issuecomment-363062924

**Comment by Keats** [comment]
@lirusaito can you check if that works for your templates or if you are still running into macro issues?

**Comment by Lirusaito** [comment]
Macro issues disappeared, I think I was chasing ghosts (if you mean rust macro issues), as for my personal macro issues, I just have the work arounds in place to deal with them.
I'll try this change tomorrow, unless I start my laptop up today, then probably today.

**Comment by Lirusaito** [comment]
@Keats I went to test it with my personal templates, but they are .html.tera files set up for rocket, is there a quick bit of code I could use to make tera consume everything as though it didn't have any extensions? Ideally, I'd like to avoid mucking up too many files just to test, since I have other uncommitted work adjacent to the areas I'll be testing in.

**Comment by Keats** [comment]
No easy way to do that no. You can cp the templates folder and do a regex
search/replace for extends/include/import to add the extension in the call

**Comment by Lirusaito** [comment]
Oki, I think I'll cp then remove extensions.

**Comment by Keats** [comment]
@Lirusaito ping

File: netlify.toml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[build]
+    base    = "docs"
+    publish = "docs/public"
+    command = "gutenberg build"
+
+[context.production.environment]
```
Comment (calavera) [inline netlify.toml]: if you don't want to use different binary versions in production and deploy previews, you can replace both `context.production.environment` and `context.deploy-preview.environment` with this single directive:

```
[build.environment]
  GUTENBERG_VERSION = "0.3.0"
```
that will apply to all contexts. If you need to use a specific version for a context, then you can override this value in the specific context.

**Comment by Lirusaito** [comment]
My apologies for the delay, I'm not quite sure what you want me to be testing, so here's a list of all the issues:
+ 256 - Flow control with nested imports: Still an issue (I see that this is marked for next major, so that makes sense)
+ 255 - Macro files with imports: No longer an issue
+ Just for the heck of it, I also tried importing inside a macro itself, and it was still an unexpected tag, which makes sense.

**Comment by Keats** [comment]
> Just for the heck of it, I also tried importing inside a macro itself, and it was still an unexpected tag, which makes sense.

I don't believe this has ever been supported.
Anyway, seems that I can do a release now, thanks!

**Pull request merged**